CHANGELOG/CHANGELOG-1.12.md

```diff
@@ -1,5 +1,26 @@
 # Changelog 1.12
 
+## 1.12.3 - 2024-01-09
+
+### Added
+
+- Update `Install-WindowsCNI-Containerd.ps1` script to make it compatible with containerd 1.7. ([#5528](https://github.com/antrea-io/antrea/pull/5528), [@NamanAg30])
+- Store NetworkPolicy in filesystem as fallback data source to let antre-agent fallback to use the files if it can't connect to antrea-controller on startup. ([#5739](https://github.com/antrea-io/antrea/pull/5739), [@tnqn])
```

Comment:
Don't know if we want to keep it under Added, or move it under Changed / Fixed?
cc @tnqn

Reply:
I feel it should be Fixed as it doesn't add anything new to users but fixes a bug by adding a mechanism to implementation. The Added section is from user's perspective.